plaidml/core/core.h

```diff
@@ -5,12 +5,32 @@
 #include <cstring>
 #include <functional>
 #include <memory>
+#include <sstream>
 #include <stdexcept>
 #include <string>
 #include <vector>
 
 #include "plaidml/core/ffi.h"
 
+#if __has_include(<source_location>)
+#include <source_location>
+using edsl_source_location = std::source_location;
+#define ERRORTRACING true
```

Comment:
Not sure this should go into the public header. It seems only be used by our own internal tests.

Reply:
Ok; I can remove this and just make replace its use in `edsl_test.cc` with a similar `__has_include` check.